doc/source/notebooks/theory/Sanity_check.ipynb

```diff
@@ -378,9 +204,9 @@
       "  kernel variance     = 0.82561\n",
       "  likelihood variance = 0.16002\n",
       "SGPR\n",
-      "  kernel lengthscale  = 1.0774\n",
-      "  kernel variance     = 0.82561\n",
-      "  likelihood variance = 0.16002\n",
+      "  kernel lengthscale  = 4.5905e-09\n",
```

Comment:
This looks significantly different from the previous answer. Did this notebook get run after #1106 was merged into develop-2.0 (which fixes a bug in the `SGPR` model)?

Reply:
Yeah, the bug is still in the branch :) I was about to fix it, but decided to wait #1106.

Reply:
:+1: ok:) I guess you'll just have to merge develop-2.0 back in then, now that the bugfix is in there:)